sensu-plugins-vsphere.gemspec

```diff
@@ -29,9 +27,8 @@ Gem::Specification.new do |s| # rubocop:disable Metrics/BlockLength
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsVsphere::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2.0'
-
   s.add_runtime_dependency 'rbvmomi',      '~> 1.11.6'
+  s.add_runtime_dependency 'sensu-plugin', '~> 2.4.0'
```

Comment:
This makes this a breaking change that needs to be called out in the changelog like this:
```
### Breaking Changes
- bumped dependency of `sensu-plugin` to 2.x you can read about it [here](https://github.com/sensu-plugins/sensu-plugin/blob/master/CHANGELOG.md#v145---2017-03-07)
```